mtest/libmscore/plugins/testTextStyle-start.mscx

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
by convention, "start" files are called file.mscx and reference files file-ref.mscx